Raven.NET.Core/Observers/RavenWatcher.cs

```diff
@@ -64,27 +85,35 @@ public void UnWatch(string name, RavenSubject subject)
         {
             var raven = _ravenProvider.GetRaven(name) as RavenWatcher;
             raven.watchedSubjects.Remove(subject);
-            
-            if(raven._ravenSettings.AutoDestroy)
+            _logger.LogInformation($"Subject {subject.UniqueId} is no longer watched by {RavenName}");
+
+            if (!raven._ravenSettings.AutoDestroy)
```

Comment:
Add brackets, I think we discussed to always have those, correct me if i am wrong

Reply:
True, will change